sycl/include/sycl/ext/oneapi/experimental/invoke_simd.hpp

```diff
@@ -40,7 +40,7 @@
 template <bool IsFunc, class SpmdRet, class SimdCallee, class... SpmdArgs,
           class = std::enable_if_t<!IsFunc>>
 SYCL_EXTERNAL __regcall SpmdRet
-__builtin_invoke_simd(SimdCallee target, const void *obj, SpmdArgs... args)
+__builtin_invoke_simd(SimdCallee &&target, const void *obj, SpmdArgs... args)
```

Comment:
As we talked, couple of comments:
- target is really not the user function, but the helper
- no need in `&&` as we know we pass by value always (from invoke_simd).
So, I suggest:
```cpp
template <bool IsFunc, class SpmdRet, class HelperFunc, class... UserSimdFuncAndSpmdArgs,
          class = std::enable_if_t<!IsFunc>>
SYCL_EXTERNAL __regcall SpmdRet
__builtin_invoke_simd(HelperFunc target, const void *obj, UserSimdFuncAndSpmdArgs... args)
```